pages/community/events/webinar.md

```diff
@@ -0,0 +1,9 @@
+---
+title: 'GraphQL to Simplify Data Access for Cloud Native'
+type: WEBINAR
+description: 'In this webinar, we’ll look at the emerging role of GraphQL as the unifying query language for federated resources in the modern, cloud-native environment. Between SDKs, REST endpoints, secondary GraphQL schemas.
```

Comment:
```suggestion
description: 'In this webinar, we’ll look at the emerging role of GraphQL as the unifying query language for federated resources in the modern, cloud-native environment.'
```

do we actually host webinars? it seems to me we host streams, not webinars.... why not have a `streams.md` file instead of this current **webinar** one? 🤔

Reply:
@alequetzalli thanks for the review, concerning the list of events, I'm getting rid of them cuz I'm only using em for the sake of view